### ISSUE TITLE
Missions et contrôles - Affichage sur la carto des missions et des contrôles

### ISSUE THREAD

**Issue by AdelineCelier** [MTES-MCT/monitorfish#1743]
Les écrans sont dans le XD du rapportage Fish, du

**Comment by AdelineCelier** [comment]
_**Revue en cours**_

**Pop-up de mission**
- Problème de chevauchement de la croix et du nom de l'unité 
- Manque la pastille de couleur à côté du statut "clôturé" (et le mot "clôturé" doit être de la couleur du statut aussi)
![image](https://user-images.githubusercontent.com/71878648/230333339-b23ea0bc-8232-45aa-9379-e733d53835d9.png)

**Comment by louptheron** [comment]
@AdelineCelier 
- Dans le cas où il n'y a ni NATINF, ni appréhension, ne pas mettre de tag et ne pas mettre de mention particulière (cf. aujourd'hui il y a une phrase "aucune appréhension") : c'était pour ne pas avoir de blanc, vu que la taille de la box est fixe
- Changer l'icône dans le cas d'une mission multi-types : je mettrais bien l'icône "more" (les trois petits points): ça demande un changement des icônes, on peut faire ça plus tard si c'est confusant ?

**Comment by AdelineCelier** [comment]
@louptheron 

> "Changer l'icône dans le cas d'une mission multi-types : je mettrais bien l'icône "more" (les trois petits points)" : ça demande un changement des icônes, on peut faire ça plus tard si c'est confusant ?
Oui, on peut faire ça plus tard, j'ai créé le ticket #1955 . Il est sur l'amélioration des icônes au global (les couleurs ne me vont décidémment pas je crois, je vais les retravailler ^^)

> "Quand on clique sur une mission, le point de contrôle ajouté précédemment ne s'affiche pas, il faut recharger la page pour qu'il s'affiche." : J'ai fait un truc rapide mais je vais faire ça mieux dans une prochaine PR (grosse tâche
Ok c'est noté, pour l'instant je ne vois pas de changement de mon côté
